packages/react/.storybook/main.js

```diff
@@ -29,7 +30,7 @@ module.exports = {
 
     // High quality 'original source' sourcemaps are slow to generate on initial builds and rebuilds.
     // Using cheap-module-eval-source-map speeds up builds and rebuilds in development while not sacrificing too much source map quality.
-    config.devtool = configType === 'DEVELOPMENT' ? 'cheap-module-eval-source-map' : 'source-map';
+    config.devtool = configType === 'DEVELOPMENT' ? 'cheap-module-eval-source-map' : '';
```

Comment:
remove the sourcemap here for production builds, so we're no un-doing the work of the turbo-build addon.